src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@import 'variables';
+@import 'header';
+@import 'nav';
+@import 'catalog';
+@import 'card';
+@import 'star';
+
+* {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use a more specific selector than the universal selector (*) to avoid unintended styling of all elements.